apps/expo/src/app/_layout.tsx

```diff
@@ -109,19 +112,19 @@ export default function RootLayout() {
     <QueryClientProvider client={queryClient}>
       <SafeAreaProvider>
         <AgentProvider value={agent}>
-          <StatusBar style="dark" />
+          <StatusBar style="auto" />
           {loading && <SplashScreen />}
           <Stack
             screenOptions={{
               headerShown: true,
               headerBackTitle: "",
               fullScreenGestureEnabled: true,
               headerStyle: {
-                backgroundColor: "#fff",
+                backgroundColor: colorScheme === "light" ? "#fff" : "#000",
               },
             }}
           />
-          <StatusBar />
+          <StatusBar style="auto" />
```

Comment:
don't think there needs to be 2 `<StatusBar>`s

Reply:
ok this is crazy but if i remove one of them the whole thing breaks, need to debug

Reply:
haha ok, I do love a load-bearing useless component